trx/tests/test_memmap.py

```diff
@@ -121,24 +126,25 @@ def test__create_memmap(basename, create, expected):
         assert np.array_equal(mmarr, expected)
 
     else:
-        mmarr = tmm._create_memmap(filename=basename, shape=(0,),
+        filename = os.path.join(mkdtemp(), basename)
+        mmarr = tmm._create_memmap(filename=filename, shape=(0,),
                                    dtype=np.int16)
-        assert os.path.isfile(basename)
+        assert os.path.isfile(filename)
         assert np.array_equal(mmarr, np.zeros(shape=(0,), dtype=np.float32))
 
 
 # need dpg test with missing keys
 @pytest.mark.parametrize(
     "path,check_dpg,value_error",
     [
-        ("trx/tests/data/small_compressed.trx", False, False),
-        ("trx/tests/data/small.trx", True, False),
-        ("trx/tests/data/small_fldr.trx", False, False),
-        ("trx/tests/data/dontexist.trx", False, True),
+        ("small_compressed.trx", False, False),
+        ("small.trx", True, False),
+        ("small_fldr.trx", False, False),
+        ("dontexist.trx", False, True),
     ],
 )
 def test__load(path, check_dpg, value_error):
```

Comment:
Does this intentionally have two underscores? 
```suggestion
def test_load(path, check_dpg, value_error):
```